src/main/groovy/life/qbic/portal/sampletracking/web/views/ControlElements.groovy

```diff
@@ -32,17 +32,13 @@ class ControlElements extends VerticalLayout {
 
     private def initLayout() {
 
-        // Add textfield showing email address of portal user
+        // Add textField showing email address of portal user
```

Comment:
This is not meant as a class name but rather a description of the functionality. Therefore no camel case needed.

Reply:
```suggestion
        // Add text showing email address of portal user
```